runtime/near-test-contracts/build.rs

```diff
@@ -25,6 +25,11 @@ fn try_main() -> Result<(), Error> {
         &["--features", "base_protocol"],
         "test_contract_rs_base_protocol",
     )?;
+    build_contract(
+        "./test-contract-rs",
+        &["--features", "protocol_feature_function_call_weight"],
+        "test_contract_rs_function_call_weight",
+    )?;
```

Comment:
Why do we need a separate blob with a new feature enabled? Why can’t this just be in test_contract_rs?

Reply:
It can be, want to clean this up in a follow up: https://github.com/near/nearcore/pull/6620